src/a4/domain/BoardSpaceFactory.java

```diff
@@ -0,0 +1,44 @@
+package a4.domain;
+
+import java.util.ArrayList;
+
+public class BoardSpaceFactory {
+	int location;
+	ArrayList<Player> players = new ArrayList<Player>();
+	static int counter = 1;
+
```

Comment:
Does the board space factory need the location, players and a counter? Isn't that what boardSpace keeps track of